File: C/Lista_02(Estrutura_Condicional)/Menor_de_Tres.c
#include <stdio.h>

int main () {

    int valor1, valor2, valor3;

    printf("Primeiro valor: ");
    scanf("%d", &valor1);

    printf("Segundo valor: ");
    scanf("%d", &valor2);

    printf("Terceiro valor: ");
    scanf("%d", &valor3);

    int menor;

    if (valor1<=valor2 && valor1<=valor3){

        menor = valor1;

    }
    else if (valor2<=valor1 && valor2<=valor3){

        menor = valor2;

    }
    else{

        menor = valor3;

    }

    printf("MENOR = %d", menor);

    return 0;

}

File: Java/Lista_02(Estrutura_Condicional)/Menor_de_Tres.java
import java.util.Scanner;

public class Menor_de_Tres {

	public static void main(String[] args) {
		
		Scanner s = new Scanner(System.in);
		
		System.out.print("Primeiro valor: ");
		int n1 = s.nextInt();
		
		System.out.print("Segundo valor: ");
		int n2 = s.nextInt();
		
		System.out.print("Terceiro valor: ");
		int n3 = s.nextInt();
		
		if(n1<=n2 && n1<=n3) {
			System.out.println("MENOR = " + n1);
		}
		else if(n2<=n1 && n2<=n3) {
			System.out.println("MENOR = " + n2);
		}
		else {
			System.out.println("MENOR = " + n3);
		}
		
		s.close();

	}

}
